src/main/java/com/redhat/cloud/policies/app/EnvironmentInfo.java

```diff
@@ -0,0 +1,26 @@
+package com.redhat.cloud.policies.app;
+
+import io.quarkus.runtime.Startup;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jboss.logging.Logger;
+
+import javax.annotation.PostConstruct;
+import java.lang.reflect.Field;
```

Comment:
This isn't used.

Reply:
Woaah... why isn't the style check complaining about it?